scripts/analytics/premiumFeatureViewTime.js

```diff
@@ -8,24 +8,21 @@ if (process.argv.length !== 4) {
 
 require('coffee-script');
 require('coffee-script/register');
-GLOBAL._ = require('lodash')
+const _ = GLOBAL._ = require('lodash');
 _.str = require('underscore.string')
 _.mixin(_.str.exports())
 
 const mongoConnUrl = process.argv[2];
 const mongoConnUrlAnalytics = process.argv[3];
-const mongoConnUrlLevelSessions = process.argv[4];
 const config = require('../../server_config');
-config.mongo.level_session_replica_string = mongoConnUrlLevelSessions;
 
 const mongoose = require('mongoose');
 const MongoClient = require('mongodb').MongoClient;
 const ObjectID = require('mongodb').ObjectID;
 const Promise = require('bluebird');
-const genstats = require('genstats');
 const co = require('co');
 const moment = require('moment');
-const LevelSession = require('../../server/models/LevelSession');
+const User = require('../../server/models/User');
 
 const startOfWeek = 5 // Friday morning at 00:00
 const today = new Date();
```

Comment:
The following section could use some comments explaining what it's trying to do. Mainly I'm wondering, will `startDay` always be a certain day of the week?